init/test-postgres.js

```diff
@@ -0,0 +1,23 @@
+const { PostgreSqlContainer } = require ('@testcontainers/postgresql');
+const { spawn } = require('node:child_process');
+
+const run = async () => {
+    const container = await new PostgreSqlContainer().start();
```

Comment:
Seems like there is a way to specify port in java https://stackoverflow.com/questions/69132686/how-can-i-set-the-port-for-postgresql-when-using-testcontainers, maybe there's a way to do it in javascript, too. But this way is also fine.

Reply:
It can be done by setting `.withExposedPorts({container: 5432, host: 5432})` but probably better to have random port to make sure on local machine there is no conflict with other application or postgres running on 5432.